packages/fxa-auth-server/docs/swagger/shared/descriptions.ts

```diff
@@ -161,7 +215,11 @@ const DESCRIPTIONS = {
     'Indicates whether a new `sessionToken` is required, default to `false`.',
   state:
     'An opaque string value provided by the connecting client application, which will be returned unmodified upon redirection alongside the authorization code. This can be used by the connecting client guard against certain classes of attack in the redirect-based OAuth flow to verify that the redirect is authentic.',
+  status:
+    'The status of the product (e.g. `active`, `canceled`, `trialing`, `unpaid`, etc).',
   sub: 'The hex id of the user.',
+  subscriptionId: 'A unique identifier for the subscription.',
+  subscriptions: 'A list of subscriptions',
```

Comment:
Worth noting this is a list of all subscriptions (web and IAP combined) based on the only two instances of this response property I could find in the code (for `/account` and `/billing-and-subscriptions` endpoints).